### PR TITLE
Add couple of roles <3

### PR DESCRIPTION
**Pull request by pooniena2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,3 +161,8 @@ This is a list of Canadian tech internships for 2023. This list is maintained by
...
+| Softworks Group             | [Database Software Engineer](https://softworksgroup.zohorecruit.com/jobs/Careers/689970000000589238/Database-Software-Engineer-Intern?source=Indeed-Sponsored)                                                                                                                                                | Remote                                                  | ???       | ???       |                 Must be PR/ Canadian Citizenship                 | 
+| Dormakaba Group             | [Cloud Software Developer Intern](https://jobs.dormakaba.com/job/Montreal-Cloud-Software-Developer-Intern-QC-QC-H4P-2G7/876189201/)                                                                                                                                                | Montreal                                                  | May 2023       | August 2023       |                                 |
```
Comment (KnlnKS) [inline README.md:167]: This company seems to have multiple roles. Could you replace the link with this: `https://jobs.dormakaba.com/search/?q=intern&q2=&optionsFacetsDD_country=CA&title=intern`.

Comment (KnlnKS) [reply README.md:167]: Additionally, instead of listing the end date as `August 2023` could you list the length of the term as `4 months`.

**Review by KnlnKS — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pooniena2) [reply README.md]: Done!

**Review by KnlnKS — APPROVED** [review]
Thanks 👍

**Pull request merged**